pkg/server/routes.go

```diff
@@ -67,6 +67,7 @@ func (s *Server) routes(
 			store.SaveSystemIntake,
 			store.FetchSystemIntakeByID,
 			services.NewAuthorizeSaveSystemIntake(s.logger),
+			services.NewValidateAndSubmitSystemIntake(cedarClient.ValidateAndSubmitSystemIntake),
```

Comment:
Related to other comment, here we just get `cedarClient.ValidateAndSubmitSystemIntake` as it has the same signature. Then mock that.